gatsby-config.js

```diff
@@ -42,17 +44,21 @@ module.exports = {
      */
     searchgov: {
       
-      // You should not change this.
+      // Only change this if you're using a CNAME. Learn more here: https://search.gov/manual/cname.html
       endpoint: 'https://search.usa.gov',
       
-      // replace this with your search.gov account
-      affiliate: 'federalist-uswds-example',
+      // Replace this with your search.gov account.
+      affiliate: 'usasearch',
```

Comment:
Please revert to `'federalist-uswds-example'`

Reply:
Updated